ecs-image-build/docker_start.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+#
+# Start script for ch.gov.uk running in ecs
+
+APP_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+PORT=${PORT:=10000}
+
+source /opt/perlbrew/etc/bashrc
+export PERL5LIB="${APP_DIR}/local/lib/perl5"
+#exec perlbrew exec perl "${APP_DIR}/script/start_app" daemon -l "http://*:${PORT}"
+/opt/perlbrew/perls/perl-5.18.2/bin/perl "${APP_DIR}/script/start_app" daemon -l "http://*:${PORT}"
```

Comment:
To avoid tight coupling with the build image tooling, can we inject the environment using `perlbrew` here rather than hard-coding the Perl interpreter path, and also replace the `bash` shell process to flatten the process hierarchy (using `exec`):

```suggestion
exec perlbrew exec env PERL5LIB="${APP_DIR}/local/lib/perl5" "${APP_DIR}/script/start_app" daemon -l "http://*:${PORT}"
```